pkgdb2/ui/packages.py

```diff
@@ -345,8 +345,18 @@ def package_anitya(package, full=True):
         package
     )
 
-    req = requests.get(url)
-    data = req.json()
+    data = {}
+    try:
+        req = requests.get(url)
+        if req.status_code != 200:
+            flask.flash(
```

Comment:
Would this be cleaner if you raised an exception (which would then be caught around line 357)?